flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/shuffle/KafkaShuffleITCase.java

```diff
@@ -61,7 +59,7 @@
 /** Simple End to End Test for Kafka. */
 public class KafkaShuffleITCase extends KafkaShuffleTestBase {
 
-    @Rule public final Timeout timeout = Timeout.millis(600000L);
+    //    @RegisterExtension public final Timeout timeout = Timeout.millis(600000L);
```

Comment:
Same here timeout should be migrated as well to junit5

Reply:
fixed it